src/pdn/README.md

```diff
@@ -164,24 +163,24 @@ define_power_switch_cell -name POWER_SWITCH -control SLEEP -switched_power VDD -
 ```
 
 ```tcl
-define_power_switch_cell 
-    -name name 
+define_power_switch_cell
     -control control_pin
-    -power_switchable power_switchable_pin
+    -ground ground_pin
+    -name name
     -power unswitched_power_pin
-    -ground ground_pin 
+    -power_switchable power_switchable_pin
     [-acknowledge acknowledge_pin_name]
 ```
 
 #### Options
 
 | Switch Name | Description |
 | ----- | ----- |
-| `-name` | The name of the power switch cell. |
 | `-control` | The name of the power control port of the power switch cell. |
-| `-switched_power` | Defines the name of the pin that outputs the switched power net. |
-| `-power` | Defines the name of the pin that connects to the unswitched power net. |
 | `-ground` | Defines the name of the pin that connects to the ground net. |
+| `-name` | The name of the power switch cell. |
+| `-power` | Defines the name of the pin that connects to the unswitched power net. |
+| `-switched_power` | Defines the name of the pin that outputs the switched power net. |
 | `-acknowledge` | Defines the name of the output control signal of the power control switch if it has one. |
```

Comment:
Do we want to group these in the detailed options list too?  The mandatory/optional distinction isn't clear and it just looks unsorted.  Alternatively use [-acknowledge] to clarify.

Reply:
With grouping it matches the usage and it's easier to see discrepancies and missing option description. I don't have a strong opinion on that though. What do you think?

Reply:
I've updated the PR with the brackets on optional flags in description.

Reply:
@gadfort I'll fixed the issues in pdn doc. Here are the ones I found;

pdngen: -verbose missing from description. I'll found one.

define_pdn_grid: -power_control_signal in usage and -power_control in description. I guess the usage one is correct

define_power_switch_cell: -power_switchable in usage and -switched_power. I guess the usage is correct too.

adn_pdn_ring: -ground_pads and -power_pads missing from description. What description would you add?

Reply:
-power_control is correct https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L864
-switched_power is correct https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L93
add_pdn_ring: -ground_pads and -power_pads these are deprecated and should not be documented.

Reply:
> -switched_power is correct
> 
> https://github.com/The-OpenROAD-Project/OpenROAD/blob/ea853cdd83be73ec2fe8bcbaebae891773cfca65/src/pdn/src/pdn.tcl#L93
> 
It was about define_power_switch_cell. I guess -power_switchable is correct then.

sta::define_cmd_args "define_power_switch_cell" {-name <name> \
                                                 -control <control_pin> \
                                                 [-acknowledge <acknowledge_pin>] \
                                                 -power_switchable <power_switchable_pin> \
                                                 -power <power_pin> \
                                                 -ground <ground_pin> }